webapp/src/main/java/com/box/l10n/mojito/service/blobstorage/redis/RedisClient.java

```diff
@@ -77,4 +80,17 @@ public void put(String key, byte[] value, Retention retention) {
       LOG.error("Unable to save key to Redis: {}", key, e);
     }
   }
+
+  public Object executeScript(RedisScript script, List<String> keys, List<String> args)
```

Comment:
For our case the output of this will always be either 0 or 1 so I think we should set return type here to Long rather than casting it everywhere else. Maybe rename this function to executeRateLimiterScript and if we need a different type of lua script output in future we can add a separate function for it